src/js/utils/BetaOptInUtil.js

```diff
@@ -0,0 +1,104 @@
+import Util from "./Util";
+
+const BETA_OPTIN_CONFIG_PATH = ["service"];
```

Comment:
I don't think that these constants actually help and would argue that it's these "options" make it even harder to reason about the respective code.